Foundation/URLSession/URLSessionConfiguration.swift

```diff
@@ -209,4 +209,19 @@ open class URLSessionConfiguration : NSObject, NSCopying {
      */
      open var protocolClasses: [AnyClass]?
 
+     /* A Boolean value that indicates whether the session should wait for connectivity to become available, or fail immediately */
+     open var waitsForConnectivity: Bool = false
```

Comment:
These should have a warning:

`@available(*,deprecated,message:"Not available on non-Darwin platforms")`